src/app/commands/cmd_sprite_size.cpp

```diff
@@ -199,9 +222,23 @@ class SpriteSizeWindow : public app::gen::SpriteSize {
     method()->addItem("Nearest-neighbor");
     method()->addItem("Bilinear");
     method()->addItem("RotSprite");
-    method()->setSelectedItemIndex(
-      get_config_int("SpriteSize", "Method",
-                     doc::algorithm::RESIZE_METHOD_NEAREST_NEIGHBOR));
+    int resize_method;
+    if (params.method.isSet()) {
+      if (params.method() == doc::algorithm::RESIZE_METHOD_BILINEAR)
+        resize_method = 1;
+      else if (params.method() == doc::algorithm::RESIZE_METHOD_ROTSPRITE)
+        resize_method = 2;
+      else
+        resize_method = 0;
+    }
+    else
+      resize_method = get_config_int("SpriteSize", "Method",
+                                     doc::algorithm::RESIZE_METHOD_NEAREST_NEIGHBOR);
+    set_config_int("SpriteSize", "Method", resize_method);
```

Comment:
We shouldn't modifying the configuration here with `set_config_int`, only getting configuration with `get_config_int`.

Reply:
Got it!